packages/@o3r/eslint-config/src/rules/typescript/import.cjs

```diff
@@ -0,0 +1,25 @@
+/**
+ * @type {import('@typescript-eslint/utils').TSESLint.FlatConfig.ConfigArray}
+ */
+const config = [
+  {
+    name: '@o3r/overrides/import',
+    files: [
+      '**/*.{c,m,}{t,j}s'
+    ],
+    rules: {
```

Comment:
We did not talked about it, but for imports I would add a trailing comma mandatory.
In this way if a new import is added at the end of the list only one line will appear on the diff.
Without trailing comma the diff will be on 2 lines.
https://eslint.style/rules/js/comma-dangle#options